app/config/AppConfig.scala

```diff
@@ -32,6 +35,10 @@ trait AppConfig {
   val ifBaseUrl: String
   val ifEnvironment: String
 
+  val useEncryption: Boolean
+  val encryptionKey: String
+  val mongoTTL: Int
```

Comment:
It's recommended to use `def` in traits. The class implementing the trait can choose to use `val`, `lazy val` or `def`

Reply:
I have amended this in the next commit